lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -38,11 +38,13 @@ function TransferListItem<T extends string | number>(props: TransferListItemProp
   );
 }
 
+TransferListItem.propTypes = transferListItemPropTypes;
```

Comment:
As an internal component `TransferListItem` does not need prop types. Only externally visible components need them.

Reply:
> As an internal component `TransferListItem` does not need prop types. Only externally visible components need them.

Ah, that makes sense. Will remove it.

Reply:
> As an internal component `TransferListItem` does not need prop types. Only externally visible components need them.

Thanks again, I've removed the TransferListItem.propTypes in this commit:
0f0c1c1799991ec9d4c9e64eca29b9c86c4061c0